### PR TITLE
Implement log icons update

### PR DESCRIPTION
**Pull request by rsasch**
- replaced status icons with clarity icons and tooltips
- combined log columns into one and replaced links with clarity icons and tooltips

### CODE REVIEW

File: ui/src/app/job-details/tasks/tasks.component.css [hunk 1]
```diff
@@ -52,3 +52,24 @@
...
+  color: #fff;
+  display: block;
+  margin: -0.6rem 0.5rem 0 0;
+}
+
+clr-icon {
+  margin-right: 1rem;
+}
+
+::ng-deep clr-icon.has-alert.log-error svg.can-alert path.clr-i-alert.clr-i-outline--alerted {
```
Comment (calbach) [inline ui/src/app/job-details/tasks/tasks.component.css]: `::ng-deep` I think should be avoided if possible. This CSS selector is also pretty complicated and I could imagine it breaking if we upgraded our clarity icons version, for example. Have you looked into whether that is any other way to apply this style?

File: ui/src/app/job-details/tasks/tasks.component.html [hunk 2]
```diff
@@ -49,20 +51,27 @@
...
-        </mat-cell>
-      </ng-container>
-      <ng-container matColumnDef="stderr">
-        <mat-header-cell *matHeaderCellDef> Stderr </mat-header-cell>
-        <mat-cell *matCellDef="let t" class="column">
-          <a class="log-item" href="{{ getResourceUrl(t.stderr) }}">
-            {{ getResourceFileName(t.stderr) }}
-          </a>
+          <clr-tooltip *ngIf="getResourceUrl(t.stdout)">
+            <a class="log-item" href="{{ getResourceUrl(t.stdout) }}">
```
Comment (calbach) [inline ui/src/app/job-details/tasks/tasks.component.html]: So the link goes into the tooltip? Should the icon also be a link? Could you provide a link to the mocks for this? Couldn't find this interaction.

File: ui/src/app/job-details/tasks/tasks.component.css (hunk 1 above)
Comment (rsasch) [reply ui/src/app/job-details/tasks/tasks.component.css]: I wasn't able to get this to work without using ::ng-deep, but I'm definitely open to not using it.  I can definitely shorten the list of selectors.

File: ui/src/app/job-details/tasks/tasks.component.html (hunk 2 above)
Comment (calbach) [reply ui/src/app/job-details/tasks/tasks.component.html]: Per offline discussion, apparently this does put the link onto the icon and not in the tooltip. I find the structure of this HTML surprising, but it appears to be the intended usage.

**Comment by calbach** [comment]
Not sure where the new Circle failure came from, might be unrelated to this.

I created a change to pin the angular-cli which should hopefully fix this.

**Pull request merged**